### PR TITLE
Feature/customize adapter

### PR DESCRIPTION
**Pull request by mikefogg**
@bploetz This adds the ability to customize the adapter.

### CODE REVIEW

File: lib/togglehq/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Togglehq
-  VERSION = "1.0.0"
+  VERSION = "1.0.1"
```
Comment (bploetz) [inline lib/togglehq/version.rb]: Revert this change. We'll do the version bump and tag in a separate commit.

File: togglehq.gemspec [hunk 2]
```diff
@@ -28,6 +28,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "simplecov"
 
   spec.add_dependency "faraday"
-  spec.add_dependency "net-http-persistent"
+  spec.add_dependency "net-http-persistent", "2.9.4"
```
Comment (bploetz) [inline togglehq.gemspec]: This is way too specific. We should allow the enclosing app define hard dependencies if they want to, but keep this version open ended.

**Comment by mikefogg** [comment]
@bploetz Fixed up!

**Comment by bploetz** [comment]
@mikefogg  LGTM. Once merged to master I'll cut a new version of the gem
